app/views/user_mailer/shared/_in_person_warning_banner.html.erb

```diff
@@ -0,0 +1,11 @@
+<table class="usa-alert-no-border usa-alert--warning margin-bottom-4 display-block">
+  <tbody>
+  <tr>
+    <td style="width:16px;padding-right:8px;">
+      <%= image_tag('email/in-person-warning.png', width: 16, height: 24, alt: '') %>
+    </td>
```

Comment:
I confirmed that we should be using png files in our email templates. I once used an svg file and it was not supported.

Reply:
I think since this is more of a decorative icon having `alt=''` is okay.

https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt#usage_notes

Reply:
We talked about this in a huddle. Thanks for the resources as well. Resolving